packages/AlgernonTests.package/ALGTest.class/instance/prevAlgernonSingletone..st

```diff
@@ -0,0 +1,3 @@
+accessing
+prevAlgernonSingletone: anObject
```

Comment:
I think there is a typo here. It should be `Singleton`

Reply:
Also we have another instance variable starting with `previous` maybe we should be consistent.